change/@fluentui-web-components-21819dfb-c067-4641-b129-18bd5f3ac496.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29924&commitId=e23bc7841144f6477719cca80d51143f4dc44d76&env=prod&clientType=FLUENTUI)


  <details><summary><b>Switch</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Switch.Checked - RTL.normal.chromium.png|97| Changed|
|Switch.Checked - RTL.hover.chromium.png|104| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>